purchase_request_to_procurement/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Purchase Request To Procurement
+===============================
+
+This module introduces the possiblity to create procurement order from purchase requets
```

Comment:
The purchase_request module is LGPL. Do you want to change to AGPL this one? I am fine with it. Just asking